Tests/SDWebImage Tests.xcodeproj/project.pbxproj

```diff
@@ -122,6 +122,7 @@
 				DA248D50195472AA00390AB0 /* Frameworks */,
 				DA248D51195472AA00390AB0 /* Resources */,
 				D6347736BDF64FC5A4D078A4 /* Copy Pods Resources */,
+				4B51E412BA3594400947AC71 /* Embed Pods Frameworks */,
```

Comment:
I imagine the change might be due to the version of Cocoapods I used to build the test project. I believe it was 0.39.0. Cocoapods adds these build phases when running 'pod install'